specs/core/0_beacon-chain.md

```diff
@@ -767,11 +771,11 @@ def compute_activation_exit_epoch(epoch: Epoch) -> Epoch:
 #### `bls_domain`
 
 ```python
-def bls_domain(domain_type: uint64, fork_version: bytes=b'\x00' * 4) -> int:
+def bls_domain(domain_type: DomainType, fork_version: bytes=b'\x00' * 4) -> Domain:
```

Comment:
I addressed this in another PR but might as well do it here to avoid the conflict. `fork_version: bytes=b'\x00' * 4` should be `fork_version: Version=Version()`

Reply:
fixed